user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+information about other menu items in the :ref:`User Menu <user-guide-getting-started-user-menu>` document.
+
+.. _user-guide-getting-started-my-user-overview:
+
+My User Page Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+This page provides you the full details of your user profile.
+
+Its *"General"* section contains the basic profile details.
+
+.. image:: ./img/intro/user_view_general.png
+
+The set of information available in the section depends on the 
+:ref:`permissions <user-guide-user-management-permissions-basic>` defined for your user. The information may include:
+
+* Your name and system information such as status (Active), login count, date and time of the last login - in the header
+
+* Your username 
+
+* Your birthday
+
+* Information about your role and place in the company structure, such as your roles in the system and business units 
+  you have access to, your API key (if any)
+  
+* Any additional details of a user defined for your organization (in the example above this is the *"Title"* field).
+
+* A link to Outlook Add-in download (for enterprise users only)
+
+* Tags assigned to your user
+
+If there is an email saved or/and a call logged in OroCRM related to you, they will be displayed in the 
+*"Activities"* section. 
+
+.. image:: ./img/intro/user_view_activities.png
+
+If there is a :ref:`task <user-guide-activities-tasks>` or a :ref:`case <user-guide-activities-cases>` assigned to you, 
+they will be displayed in corresponding grids of the "*Additional"* section.
+
+.. image:: ./img/intro/user_view_additional.png 
+
+
+Actions Available from the "My User" Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use Action Buttons
+""""""""""""""""""
+
+Action buttons are available in the top left corner of *"My User"* page. The set of buttons depends on your organization 
+settings and user permissions. These are the same actions that you can perform in the system with regard to other users.
+Most of the actions is described in the :ref:`Productivity Tools section <user-guide-productivity-tools>`.
+
+.. image:: ./img/intro/user_view_actions.png 
+
+
+Add and Edit Tags
+"""""""""""""""""
+
+If you a permitted, you can add and edit tags for your user. The tags can be used for the search, reporting, 
+segmentation and filtering.
+
+- Hover the mouse over the *"Tags"* field in the *"General"* section of the *"My User"* page as shown below.
+
+  |
+
+  |IntroTags|
+
+  |
+  
+- Click the |IcEdit| icon, and the tag field will appear.
+
+  |
+  
+  |IntroTags1|
+
+  |
+  
+- Start entering a tag and if an existing tag is found, it will be displayed for selection. If not, a new tag will be 
+  created.  
+
+- If you have entered a new tag, it will appear in the selector with a *"(New tag)"* label. Select this to add to your 
+  user as well as the list of available tags for future use. This way you can add one or more tags that contain one or 
+  several words.
```

Comment:
Edit: If you have entered a new tag, it will appear in the selector with a *"(New tag)"* label. Select this to add it to your user account, as well as to the list of available tags for future use. This way, you can add one or more tags that contain one or several words.